lib/validator.js

```diff
@@ -78,9 +78,7 @@ Validator.prototype._convertToJsonSchema = function(parameters) {
                 // 'required' array must have at least one element according to json-schema spec,
                 // se we can't preinitialize it.
                 schema.required = schema.required || [];
-                if (schema.required.indexOf(inMapping[param.in]) < 0) {
-                    schema.required.push(inMapping[param.in]);
-                }
+                schema.required.push(inMapping[param.in]);
```

Comment:
Is this relying on each parameter only being defined once? Was the previous deduplication unnecessary?